lib/injector.rb

```diff
@@ -0,0 +1,11 @@
+class Array
+
+  def injector &lambda
```

Comment:
Unused method argument - `lambda`. If it's necessary, use `_` or `_lambda` as an argument name to indicate that it won't be used. You can also write as `injector(*)` if you want the method to accept any arguments but don't care about them.